packages/core/types/zapier.custom.d.ts

```diff
@@ -138,16 +139,20 @@ type DehydrateFunc = <T>(
 export interface ZObject {
   request: {
     // most specific overloads go first
-    (
```

Comment:
@tkcranny I wonder if the default better be `unknown`, so that we don't effectively disable type checking for all downstream code.